packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/details/details.html

```diff
@@ -1,10 +1,10 @@
 <div class="telecom-telephony-line-phone-informations">
     <header>
-        <tuc-section-back-link
-            data-tuc-section-back-link-to-state="telecom.telephony.billingAccount.line.dashboard.phone"
-            data-tuc-section-back-link-title="{{ 'telephony_group_line_phone_section_back_link' | translate }}"
+        <oui-back-button
+            sref="telecom.telephony.billingAccount.line.dashboard.phone"
```

Comment:
:warning: `sref` attribute doesn't exist.

see: https://github.com/ovh/ovh-ui-kit/tree/master/packages/components/back-button#api

Reply:
i've fix it with an href, thanks :)